CalculatorWithTests/Calculator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+
+
+
+namespace CalculatorWithTests
+{
+    public class Calculator
+    {
+       public int Add(int number1, int number2)
+        {
+            return (number1 + number2);
+        }
+
+
+        public double Subtract(double minuend,  double subtrahend)
+        {
+            return(minuend-subtrahend);
+        }
+
+        public long  Multiply(long factor1,long factor2)
+        {
+            return(factor1 * factor2);
+        }
+
+        public double Divide(double num1, double num2)
```

Comment:
The names of the parameters could be better, indicating which number is to be divided by which.